apps/examples/setjmp/setjmp_longjmp_test.c

```diff
@@ -0,0 +1,161 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <setjmp.h>
+#include <stdio.h>
+#include <unistd.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define VERIFY_OR_RETURN(value, msg, ...) \
```

Comment:
Please check coding rules.

Reply:
What do I need to fix? Where is the rulebook/code guidelines?

> ./os/tools/formatter.sh apps/examples/setjmp/setjmp_longjmp_test.c
Unchanged  apps/examples/setjmp/setjmp_longjmp_test.c
> git status
On branch setjmp_tests
nothing to commit, working directory clean

Which says the automated tool from TizenRT repository has no problems with my code.
How am I supposed to know what rule the code does not conform to?

Reply:
Oh. The formatter.sh doesn't fix violations for definitions many times.
Coding rule is [[here]](https://wiki.tizen.org/Native_Platform_Coding_Idiom_and_Style_Guide#C_Coding_Style).

You should use a tab for indentation, not spaces.

Reply:
We got that tools from SE, but actually it has some problems.